app/src/main/java/com/losandroides/learn/presentation/MainViewModel.kt

```diff
@@ -27,12 +27,19 @@ class MainViewModel @Inject constructor(
 
     private fun getItems() {
         viewModelScope.launch(dispatcher) {
-            val items = itemsUseCase()
-            _viewState.value = ViewState.Content(items)
+            itemsUseCase()
+                .fold(
+                    {
+                        _viewState.value = ViewState.Error
+                    },
+                    { items ->
+                        _viewState.value = ViewState.Content(items)
+                    })
         }
     }
 
     sealed class ViewState {
+        object Error : ViewState()
```

Comment:
Could be improved with some information inside the Error. But maybe in another STRY.

Reply:
What's does "STRY" means? :slightly_smiling_face:

Reply:
Sorry, is an acronymous of STORY. So, what I mean is we can do it in another task, and don't in this one.